src/ProductConstructionService/ProductConstructionService.WorkItems/ReminderManagerFactory.cs

```diff
@@ -21,8 +21,9 @@ public ReminderManagerFactory(IWorkItemProducerFactory workItemProducerFactory,
         _cacheFactory = cacheFactory;
     }
 
-    public IReminderManager<T> CreateReminderManager<T>(string key) where T : WorkItem
+    public IReminderManager<T> CreateReminderManager<T>(string actorId) where T : WorkItem
```

Comment:
The class and method name is quite generic, if we're only going to use it for actors

Reply:
Ah, yeah, no this should stay as `key`, I have too many branches going